python/Ganga/Utility/execute.py

```diff
@@ -194,7 +195,7 @@ def execute(command,
         python_setup (str): A python command to be executed beore the main command is
         eval_includes (str): An string used to construct an environment which, if passed, is used to eval the stdout into a python object
         update_env (bool): Should we update the env being passed to what the env was after the command finished running
-        return_code (int): This is the returned code from the command which is executed
+        return_code (list): This has element 0 modified to be the returned code from the command which is executed
```

Comment:
I think that makes sense. We'll make the fix in another PR soon. There's only a few places that call `execute()` directly so it shouldn't be too invasive.